support-frontend/assets/pages/digital-subscription-landing/components/productBlockB/productBlockB.scss

```diff
@@ -149,16 +163,51 @@ $content-padding: 10px;
 }
 
 // This is a override for "#digital-subscription-landing-page-int .product-block .component-grid-image"
-#digital-subscription-landing-page-int
-.product-block #product-block__right-content
-img.component-grid-image {
+// #digital-subscription-landing-page-int
```

Comment:
I think we can delete this comment and the one above?

Reply:
👍